package.json

```diff
@@ -10,6 +10,8 @@
   "homepage": "./",
   "license": "MIT",
   "scripts": {
+    "build-seed": "cd ./src/server-browser-scripts && tsc",
+    "run-seed": "node -e 'import(\"./dist/seed/src/server-browser-scripts/seed-data.js\").then(x=> x.resetDeals())' -r dotenv/config . dotenv_config_path=./.env",
```

Comment:
I would prefer to use the other naming ef9a5a15861de78e24690b4d7ae98d1d6975e221 
`seed-build`
`seed-run`
and to put these scripts further down

if not, can you update the README.md?